src/components/questionTwo/QuestionTwo.js

```diff
@@ -0,0 +1,355 @@
+      <p>
+        Predicted CO2 eq. emissions by 2030 (target year):{' '}
+        <strong>803&nbsp;Mt&nbsp;-&nbsp;843&nbsp;Mt</strong>
+      </p>
+    </Section>
+
+    <Section title="Predictions: Post-COVID">
+      <p>
+        Below, we show the Climate Action Tracker greenhouse gas emissions predictions
+        which were computed after the start of COVID-19 (once some effects were seen) for
+        each of the three countries in question.
+      </p>
+      <p>
+        Within these predictions, we are, once again, interested in the predicted range
+        (in megatonnes CO2 eq.) of greenhouse gas emissions for each country by that
+        country's target year.
+      </p>
+
+      <h4>
+        United States of America's Prediction<sup>[9]</sup>
+      </h4>
+      <Image src="./images/question-two/CAT_Predictions_USA_2020Jul.PNG" />
+      <p>
+        Predicted CO2 eq. emissions by 2025 (target year):{' '}
+        <strong>5,744&nbsp;Mt&nbsp;-&nbsp;6,135&nbsp;Mt</strong>
+      </p>
+
+      <h4>
+        Canada's Prediction<sup>[10]</sup>
+      </h4>
+      <Image src="./images/question-two/CAT_Predictions_Canada_2020Sep.PNG" />
+      <p>
+        Predicted CO2 eq. emissions by 2030 (target year):{' '}
+        <strong>603&nbsp;Mt&nbsp;-&nbsp;630&nbsp;Mt</strong>
+      </p>
+
+      <h4>
+        Mexico's Prediction<sup>[11]</sup>
+      </h4>
+      <Image src="./images/question-two/CAT_Predictions_Mexico_2020Sep.PNG" />
+      <p>
+        Predicted CO2 eq. emissions by 2030 (target year):{' '}
+        <strong>774&nbsp;Mt&nbsp;-&nbsp;852&nbsp;Mt</strong>
+      </p>
+    </Section>
+
+    <Section title="Calculations & Results">
+      <p>
+        The below table outlines the predicted megatonnes of C02 eq. emissions pre- and
+        post-COVID for each of the countries, and the total for the three countries.
+        Additionally, we calculate the percent difference in predicted greenhouse gas
+        emissions for each country and for all three countries.
+      </p>
+      <table>
+        <thead>
+          <th>Country</th>
+          <th>Pre-COVID</th>
+          <th>Post-COVID</th>
+          <th>Percent Difference</th>
+          <th>Target</th>
+        </thead>
+        <tbody>
+          <tr>
+            <td>USA</td>
+            <td>6,273&nbsp;Mt&nbsp;-&nbsp;6,373&nbsp;Mt</td>
+            <td>5,744&nbsp;Mt&nbsp;-&nbsp;6,135&nbsp;Mt</td>
+            <td>-8.80%</td>
+            <td>5,744&nbsp;Mt</td>
+          </tr>
+          <tr>
+            <td>Canada</td>
+            <td>630&nbsp;Mt&nbsp;-&nbsp;763&nbsp;Mt</td>
+            <td>603&nbsp;Mt&nbsp;-&nbsp;630&nbsp;Mt</td>
+            <td>-4.38%</td>
+            <td>523&nbsp;Mt</td>
+          </tr>
+          <tr>
+            <td>Mexico</td>
+            <td>803&nbsp;Mt&nbsp;-&nbsp;843&nbsp;Mt</td>
+            <td>774&nbsp;Mt&nbsp;-&nbsp;852&nbsp;Mt</td>
+            <td>-3.68%</td>
+            <td>763&nbsp;Mt</td>
```

Comment:
I know... This is ugly... But it's the easiest way to prevent `startRange - endRange` from being split with a newline.

Reply:
`&nbsp;` stands for non breaking space, it means it will insert the normal space character, but prevent the browser from splitting the line (inserting a newline) in place of the space.